source/advanced-topics.adoc

```diff
@@ -4,6 +4,187 @@
 This chapter covers topics that are beyond the scope of the introductory
 tutorial but are often useful in real-world RPM packaging.
 
+[[Signing-Packages]]
+== Signing Packages
+
+Signing a package is a way to secure the package for an end user. Secure transport can be achieved with implementation of the HTTPS protocol, which can be done when the package was downloaded just before installing. However, the packages are often downloaded in advance and stored in local repositories before they are used. The packages are signed to make sure no third party can alter the content of a package.
+
+There are three ways to sign a package:
+
+* xref:Adding-a-Signature-to-a-Package[Adding a signature to an already existing package].
+
+* xref:Replacing-a-Package-Signature[Replacing the signature on an already existing package].
+
+* xref:Build-time-signing[Signing a package at build-time].
+
+[[Adding-a-Signature-to-a-Package]]
+=== Adding a Signature to a Package
+
+In most cases packages are build without a signature. The signature is added just before release of the package.
+
+In order to add another signature to the package use the --addsign option.
```

Comment:
``package use the --addsign option.`` > ``package, use the `--addsign` option.``